packages/nodes-base/nodes/Html/Html.node.ts

```diff
@@ -0,0 +1,362 @@
+import cheerio from 'cheerio';
+import {
+	INodeExecutionData,
+	IExecuteFunctions,
+	INodeType,
+	INodeTypeDescription,
+	IDataObject,
+	NodeOperationError,
+} from 'n8n-workflow';
+import { placeholder } from './placeholder';
+import { getResolvables, getValue, makeExecutionData } from './utils';
+import type { IValueData } from './types';
+
+export class Html implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'HTML',
+		name: 'html',
+		icon: 'file:html.svg',
+		group: ['transform'],
+		version: 1,
+		subtitle: '={{ $parameter["operation"] }}',
+		description: 'Work with HTML',
+		defaults: {
+			name: 'HTML',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		parameterPane: 'wide',
+		properties: [
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				noDataExpression: true,
+				options: [
+					{
+						name: 'Generate HTML Template',
+						value: 'generateHtmlTemplate',
+						action: 'Generate HTML template',
+					},
+					{
+						name: 'Extract HTML Content',
+						value: 'extractHtmlContent',
+						action: 'Extract HTML Content',
+					},
```

Comment:
Anything related to `extractHtmlContent` was moved from the Extract HTML node but otherwise left the same.